src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -413,7 +413,7 @@ public void fireFileFinished(String fileName) {
     }
 
     @Override
-    public void finishLocalSetup() throws CheckstyleException {
+    protected void finishLocalSetup() throws CheckstyleException {
```

Comment:
maven-checkstyle-plugin and eclipse-cs plugin does not use this method, so I will not mark it as bracking compatibility.